packages/pwa/app/components/image-gallery/index.jsx

```diff
@@ -101,7 +102,19 @@ const ImageGallery = ({imageGroups = [], selectedVariationAttributes = {}, size}
             {heroImage && (
                 <Box {...styles.heroImageGroup}>
                     <AspectRatio {...styles.heroImage} ratio={1}>
-                        <Img alt={heroImage.alt} src={heroImage.disBaseLink} />
+                        <Img
+                            {...getImageProps({
```

Comment:
Nit: I'd prefer we avoid using a spread operator for functions like this. It makes the component props a bit too long. Can we create a variable `heroImageProps` = `getImageProps()` above and use it here instead?